tools/ColorTool/ColorTool/Program.cs

```diff
@@ -480,6 +481,10 @@ static void Main(string[] args)
                     case "--current":
                         PrintTable();
                         return;
+                    case "-e":
+                    case "--errors":
+                        reportErrors = true;
+                        break;
```

Comment:
Please add the switch information to the help in Resources.resx

Reply:
done

Reply:
Great, thanks

---
In reply to: [260868092](https://github.com/Microsoft/console/pull/378#discussion_r260868092) [](ancestors = 260868092)